www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Menu/Menu.xml

```diff
@@ -3,6 +3,7 @@
     <Nginx cssClass="fa fa-shield fa-fw">
       <Configuration url="/ui/nginx" />
       <Logs url="/ui/nginx/index/logs" />
+      <Banned url="/ui/nginx/index/ban" />
```

Comment:
"Banned" feels misplaced. Logs should be visible as "Log File" and probably go last...

Reply:
ban is really a ban - it is a list of banned IPs in a special alias which can be referenced by PF rules. This page does not handle log files.